routes/listingRoute.js

```diff
@@ -1,113 +1,74 @@
 import express from "express";
-import Listing from "../schemas/listingSchema.js"
 import wrapAsync from "../utils/wrapAsync.js";
-import Review from "../schemas/reviewSchema.js";
 import { isAuthenticated } from "../middlewares/isAuthenticated.js";
 import { isUserListingOwner, isUserReviewAuthor } from "../middlewares/isAuthorized.js";
 import { validateListing } from "../middlewares/validateListing.js";
 import { validateReview } from "../middlewares/validateReview.js";
+import {
+  listingIndexPage,
+  showNewListing,
+  showListing,
+  createNewListing,
+  showEditListing,
+  editListing,
+  deleteListing,
+  createReview,
+  deleteReview
+} from "../controllers/listingController.js";
+import multer from "multer";
+import { storage } from "../cloudConfig.js";
 
 const listingRoute = express.Router();
+//multer is node.js middleware for handling multipart/form-data (specially upload media)
+const upload = multer({ storage });
 
-listingRoute.get("/",
-  wrapAsync(async (req, res) => {
-    const allListings = await Listing.find({});
-    res.render("listings/index.ejs", { allListings });
-  }));
+listingRoute
+  .route("/")
```

Comment:
can use the indexRoute exported from Routes.js file of constant

Reply:
will do ✅